### DISCUSSION TITLE
bfe allows DSR architecture?

### DISCUSSION THREAD

**Discussion by curiosport** [bfenetworks/bfe#928] [Q&A (问题和回答)] [answered]
bfe allows to do this?

[https://www.loadbalancer.org/blog/content/images/2020/05/concept-dsr.gif](https://www.loadbalancer.org/blog/content/images/2020/05/concept-dsr.gif)

**Comment by mileszhang2016** [comment, marked as answer]
BFE is  a  layer-7 load balancer.
TCP connection from the client is terminated at BFE. 
And BFE will set up a new TCP connection to the RS.
So it is impossible for BFE to support DSR.
